src/main/java/edu/greenblitz/robotName/subsystems/Gyros/ReplayGyro.java

```diff
@@ -1,40 +1,42 @@
 package edu.greenblitz.robotName.subsystems.Gyros;
 
-public class ReplayGyro implements IGyro{
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ReplayGyro implements IBaseGyro {
```

Comment:
This class `override`s lots of functions just to do nothing, or worse, return null 🤢 

Consider using the `default` keyword for the interface methods and having them raise a `NotImplementedException` or something like that

Reply:
i changed a bit the logic that it wont return null